controllers/v1/assessmentsController.js

```diff
@@ -432,4 +435,182 @@ module.exports = class Assessments {
         });
     }
 
+     /**
+    * @api {get} /assessment/api/v1/assessments/metaForm/:solutionId?type="individual/institutional" Assessment Solution Meta Form
+    * @apiVersion 1.0.0
+    * @apiName Assessment Solution Meta Form
+    * @apiGroup Assessments
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiSampleRequest /assessment/api/v1/assessments/metaForm/5ed5ec4dd2afa80d0f616460?type=individual
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    * "result": [
+        {
+            "field": "name",
+            "label": "Title",
+            "value": "",
+            "visible": true,
+            "editable": true,
+            "validation": {
+                "required": true
+            },
+            "input": "text"
+        }
+    ]
+    */
+
+     /**
+   * Assessment meta form.
+   * @method
+   * @name metaForm
+   * @param {Object} req -request Data.
+   * @returns {JSON} - Assessment meta form.
+   */
+
+   async metaForm(req) {
+
+    return new Promise(async (resolve, reject) => {
+
+        try {
+
+            let assessmentForm = 
+            await assessmentsHelper.metaForm(
+                req.params._id,
+                req.query.type
+            );
+
+            return resolve(assessmentForm);
+
+        } catch (error) {
+            return reject({
+                status: error.status || httpStatusCode.internal_server_error.status,
+                message: error.message || httpStatusCode.internal_server_error.message,
+                errorObject: error
+            });
+        }
+    });
+   }
+
+   /**
+     * @api {post} /assessment/api/v1/assessments/create/:solutionId?type="individual/institutional" Create assessment solution
+     * @apiVersion 1.0.0
+     * @apiName Create assessment solution
+     * @apiGroup Assessments
+     * @apiParamExample {json} Request-Body:
+     *  {
+     * "data" : [
+     * {
+     * "field": "name",
+     * "label": "Name",
+     * "value": "Aman",
+     * "visible": true,
+     * "editable": true,
+     * "input": "text",
+     * "validation": {
```

Comment:
@aman-tunerlabs - Why are these fields required in request body ?

Reply:
@akash-above  i have not get any body params from frontend side and i have send this sample to frontend team . Thats why i assume this field will come .

Reply:
@aman-tunerlabs - Isn't the normal agreed-upon thing that, they will use the field element and against that give the value in return?

Reply:
@akash-above No man.

Reply:
@aman-tunerlabs - Did you check the current request body for observation creation along with the form for the same? If the form keys are not fixed how can we even develop the rest of the API ?